css/style.css

```diff
@@ -0,0 +1,152 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-size: 60%;
+}
+
+body{
+    font-family:'Times New Roman', Times, serif;
+    background-color: #2b2d42;
+}
+
+a{
+    text-decoration: none;
+    font-weight: 400;
+    /* 1.6*10 = 16p */
+    font-size: 2rem;
```

Comment:
defining a font size relative to root percantage, I think if you used a fixed value is better you will have more controll on your component for example if the root value is 16px that means 2rem will equal 32px